BrowserKit/Sources/Common/Utilities/FontStyle.swift

```diff
@@ -0,0 +1,58 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import UIKit
+
+// This class contains the Firefox iOS type styles as part of our design system
+public struct FXFontStyles {
```

Comment:
nit; The name of the file should match the name of the struct

Reply:
updated!